Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -500,6 +500,26 @@ python::tuple GetMolFrags(const ROMol &mol, bool asMols, bool sanitizeFrags) {
   return python::tuple(res);
 }
 
+python::list wrapAssignStereochemistry(ROMol &mol, bool cleanIt, bool force, bool flagPossibleStereoCenters, bool onlyStereoAny) {
```

Comment:
I'd prefer that this (and the other next function) return a `python::tuple`.
You'll need to change the return type here and then see below for how to do the type conversion.

Reply:
You also should add a test to the python test suite (in `rough_test.py`) for this function and the next.